src/pages/signin/LoginForm.js

```diff
@@ -65,165 +65,143 @@ const defaultProps = {
     blurOnSubmit: false,
 };
 
-class LoginForm extends React.Component {
-    constructor(props) {
-        super(props);
-        this.onTextInput = this.onTextInput.bind(this);
-        this.validateAndSubmitForm = this.validateAndSubmitForm.bind(this);
-
-        this.state = {
-            formError: false,
-            login: '',
-        };
-
-        if (this.props.account.errors || this.props.account.isLoading) {
-            Session.clearAccountMessages();
-        }
-    }
-
-    componentDidMount() {
-        if (!canFocusInputOnScreenFocus() || !this.input || !this.props.isVisible) {
-            return;
-        }
-        this.input.focus();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!prevProps.blurOnSubmit && this.props.blurOnSubmit) {
-            this.input.blur();
-        }
-        if (prevProps.isVisible || !this.props.isVisible) {
-            return;
-        }
-        this.input.focus();
-    }
+function LoginForm({account, blurOnSubmit, closeAccount, isVisible, network, translate}) {
```

Comment:
i think for code requirements we do not use destructed props - you should use here function LoginForm(props)

Reply:
Hi, @narefyev91 
Updated to use the structured props.